localstack/services/s3/s3_listener.py

```diff
@@ -320,6 +325,30 @@ def add_accept_range_header(response):
         response.headers['accept-ranges'] = 'bytes'
 
 
+def is_object_expired(path):
+    object_expiry = get_object_expiry(path)
+    timezone_from_time = str(object_expiry)[len(str(object_expiry)) - 3:]
```

Comment:
This timezone parsing (based on stripping off the string suffix) looks a bit fragile. Can we either
* standardize on GMT, and generally store and compare all timestamps as GMT/UTC timestamps?

OR

* use one of the utilities provided in the Python standard library, such that we don't need to manually strip off the timezone from the time string here?